README.md

```diff
@@ -172,6 +172,14 @@ export default {
 };
 ```
 
+### pages
+
+The directory to write prerendered pages to. It defaults to `static`.
+
+### assets
+
+The directory to write static assets (the contents of `static`, plus client-side JS and CSS generated by SvelteKit) to. Ordinarily this should be the same as pages, and it will default to whatever the value of `pages` is, but in rare circumstances you might need to output pages and assets to separate locations.
```

Comment:
It might be irrelevant if we collapse these to one option, but "it will default to whatever the value of `pages` is" is not accurate with the current implementation